src/init.cpp

```diff
@@ -1043,6 +1043,40 @@ bool AppInit2(boost::thread_group& threadGroup)
             }
         }
 
+        if (GetBoolArg("-resync", false)) {
```

Comment:
This new command-line parameter will need to be documented further up as a `HelpMessageOpt()` or it will fail the documentation check.